api/src/main/java/uk/gov/dhsc/htbhf/hmrc/factory/EligibilityResponseFactory.java

```diff
@@ -1,33 +1,50 @@
 package uk.gov.dhsc.htbhf.hmrc.factory;
 
+import org.springframework.stereotype.Component;
 import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
 import uk.gov.dhsc.htbhf.hmrc.entity.Child;
 import uk.gov.dhsc.htbhf.hmrc.entity.Household;
+import uk.gov.dhsc.htbhf.hmrc.model.ChildDTO;
 import uk.gov.dhsc.htbhf.hmrc.model.EligibilityResponse;
 
 import java.time.LocalDate;
+import java.util.List;
 import java.util.Set;
+import java.util.stream.Collectors;
 
+@Component
 public class EligibilityResponseFactory {
 
-    public static EligibilityResponse createEligibilityResponse(Household household, EligibilityStatus status) {
+    public EligibilityResponse createEligibilityResponse(Household household, EligibilityStatus status) {
         return EligibilityResponse.builder()
                 .eligibilityStatus(status)
                 .numberOfChildrenUnderOne(getNumberOfChildrenUnderOne(household.getChildren()))
                 .numberOfChildrenUnderFour(getNumberOfChildrenUnderFour(household.getChildren()))
                 .householdIdentifier(household.getHouseholdIdentifier())
+                .children(getChildrenUnderFour(household.getChildren()))
                 .build();
     }
 
-    private static Integer getNumberOfChildrenUnderOne(Set<? extends Child> children) {
+    private List<ChildDTO> getChildrenUnderFour(Set<Child> children) {
+        return children.stream()
+                .filter(child -> isUnderFour(child.getDateOfBirth()))
+                .map(child -> ChildDTO.builder().dateOfBirth(child.getDateOfBirth()).build())
+                .collect(Collectors.toList());
+    }
+
+    private boolean isUnderFour(LocalDate dateOfBirth) {
+        return dateOfBirth.isAfter(LocalDate.now().minusYears(4));
```

Comment:
Is there any value in refactoring `LocalDate.now().minusYears(4)` into a separate method as its used again on line 48? I'm not sure if caching `LocalDate.now()` would provide any benefit.

Reply:
It was mostly done because it made the filter on line 30 read nicer.